engines/dfc_provider/lib/inspect.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module DfcConnectorInspect
+  # Override PP method, which is used in pretty_inspect for rails console
+  def pretty_print_instance_variables
+    instance_variables.reject{ |var| var == :@semanticPropertiesMap }
+  end
+end
+
+# Include on all connector classes
+DataFoodConsortium::Connector.constants.each do |klass|
+  DataFoodConsortium::Connector.const_get(klass).class_eval do
```

Comment:
I think that there are more classes in that module. Isn't this only for the data types?

You could iterate through `DataFoodConsortium::Connector::Importer.type_map.keys` to patch all types recognised by the importer.